packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -208,31 +237,11 @@ export class VaBreadcrumbs {
     if (this.uswds) {
       if (!this.breadcrumbList?.length) return;
 
-      let potentialBreadcrumbs;
-
-      if (Array.isArray(this.breadcrumbList)) {
-        potentialBreadcrumbs = this.breadcrumbList;
-      } else if (typeof this.breadcrumbList === 'string') {
-        try {
-          potentialBreadcrumbs = JSON.parse(this.breadcrumbList);
-        } catch (e) {
-          return;
-        }
-      } else return;
+      let potentialBreadcrumbs = this.validateBreadcrumbs(this.breadcrumbList);
 
-      // Now validate that potentialBreadcrumbs is an array of objects with the properties "label" and "href"
-      // eslint-disable-next-line i18next/no-literal-string
-      if (
-        Array.isArray(potentialBreadcrumbs) &&
-        potentialBreadcrumbs.every(
-          item =>
-            typeof item === 'object' &&
-            item.hasOwnProperty('label') &&
-            item.hasOwnProperty('href'),
-        )
-      ) {
-        this.formattedBreadcrumbs = potentialBreadcrumbs;
-        this.updateBreadCrumbList(this.formattedBreadcrumbs);
+      console.log('potentialBreadcrumbs:', potentialBreadcrumbs);
```

Comment:
@Andrew565 - was this console.log only for debugging?

Reply:
Yep!